src/repositories/UserRepository.ts

```diff
@@ -86,13 +87,21 @@ export class UserRepository extends AbstractRepository<UserModel> {
       .createQueryBuilder("user")
       .where("user.username = :username", { username })
       .getOne();
-    if (await existingUser) throw Error('UserModel with same username already exists!');
+    if (await existingUser) {
+      if (username !== user.username) {
+        throw new ConflictError('UserModel with same username already exists!');
+      }
+    }
 
     existingUser = this.repository
       .createQueryBuilder("user")
       .where("user.venmoHandle = :venmoHandle", { venmoHandle })
       .getOne();
-    if (await existingUser) throw Error('UserModel with same venmo handle already exists!');
+      if (await existingUser) {
+        if (venmoHandle !== user.venmoHandle) {
```

Comment:
I can see logistical issues with this in the future - what if two users on Resell want to link a single venmo handle (maybe they're running a business and are selling different things). If this isn't allowed on Resell then this is fine, but I can see a possible issue in the future where someone uses someone else's Venmo handle and they can't create an account because of that (maybe it could even be a typo coincidentally). I understand the new to enforce this for a username because it's your identifier on this app but there is no way for you to verify that this venmo handle is actually your own. This isn't a high priority issue, you would probably have to handle this over an email conversation and some manual endpoint calls, but something to keep in mind.